### PR TITLE
Re-uploaded deleted pricing select component

### PR DESCRIPTION
**Pull request by ThinkDevStudios**
accidentally included the pricing select component to this branch and deleted it before being merged to master. Thus deleting the merged pricing select component.

## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

**Comment by thinkverse** [comment]
Will need @tnylea input before merging this. Since this includes `script` with AlpineJS.
If AlpineJS is going to be supported in components should it be added by Tails or per component? 🤔
Could per component cause issues if multiple versions are loaded in?

**Comment by ThinkDevStudios** [comment]
I agree. That was what I'm worried about.

File: components/pricing-select.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+                        <svg x-show="toggle != 'basic'" class="w-8 text-gray-700 stroke-current stroke-2" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+                        viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+                            <ellipse style="fill:#FFFFFF;" cx="256" cy="256" rx="256" ry="255.832"/>
+                        </svg>
+                    </div>
+                    <div class="flex-grow px-0 py-5 md:px-5">
+                        <div class="text-2xl">Basic</div>
+                        <div class="pt-1 text-sm">Start pack + 3 submissions</div>
+                    </div>
+                    <div class="self-center flex-shrink w-32 px-5 py-5 text-2xl text-center text-white md:px-5">$ 5</div>
```
Comment (thinkverse) [inline components/pricing-select.html]: `text-white` causes issues when package is not selected.

```suggestion
                    <div class="self-center flex-shrink w-32 px-5 py-5 text-2xl text-center md:px-5">$ 5</div>
```
#### Before

Package price not visible

#### After

Package price now visible

File: components/pricing-select.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
+<script src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.x.x/dist/alpine.min.js" defer></script>
```
Comment (thinkverse) [inline components/pricing-select.html]: Not needed, Tails already have Alpine loaded. 🙂
```suggestion
```

**Comment by thinkverse** [comment]
Apparently, AlpineJS is already included in Tails. 👍
It even says so in the subheading "_Some require AlpineJS._", guess we both missed that. 😂

**Pull request merged**